source/sass/donate/donate-main.scss

```diff
@@ -48,10 +51,6 @@ section {
   display: block;
 }
 
-img {
-  max-width: 100%;
-}
-
```

Comment:
Removing this to avoid redundancy, as importing `resets.scss` applies both `max-width:100%` and `height:auto` to `img`.